koku/providers/azure/provider.py

```diff
@@ -99,9 +106,15 @@ def cost_usage_source_is_reachable(self, credential_name,
             storage_accounts = azure_client.storage_client.storage_accounts
             storage_account = storage_accounts.get_properties(resource_group,
                                                               storage_account)
+            azure_service = AzureService(subscription_id, tenant_id, client_id,
+                                         client_secret, resource_group, storage_account)
```

Comment:
```suggestion
            azure_service = AzureService(**credential_name, resource_group_name=resource_group,
                                          storage_account_name=storage_account)
```
Instead of unpacking the `credential_name` dictionary (lines 94-97), you _could_ do this, but you don't have to if you don't want to.

Reply:
I added your suggestion @maskarb